kotlin/workflow-core/src/main/java/com/squareup/workflow/WorkflowContext.kt

```diff
@@ -177,6 +181,59 @@ inline fun <StateT : Any, OutputT : Any, reified T> WorkflowContext<StateT, Outp
     handler
 )
 
+/**
+ * Subscribes to [flow] and invokes [handler] when it emits the next value.
+ *
+ * This allows us to use this method to, for example, monitor a stream of values emitted by a single
+ * fow over a series of state transitions. (Use [key] if you need to work simultaneously with
+ * several [Flow]s of the same type.)
+ * The subscription will be disposed when `compose` returns without passing a [Flow] of the same
+ * type and with the same key to this method.
+ *
+ * ## Experimental Status
+ *
+ * The [Flow] API is still marked experimental, and the compiler will warn you if you use it without
```

Comment:
Warn or error? Is there a way to make it an error without opting in? I know that we can make all warnings errors, but this isn't feasible in POS.

Reply:
Just read the previous conversation with Ray. It also sounds safer to me to have a separate library given that most people ignore warnings. If Flow would be in a further development stage I wouldn't mind, but since it's the first alpha release it might be dangerous.